libp2p/protocols/pubsub/rpc/messages.nim

```diff
@@ -45,3 +46,14 @@ type
       subscriptions*: seq[SubOpts]
       messages*: seq[Message]
       control*: Option[ControlMessage]
+
+func shortLog*(m: RPCMsg): string =
```

Comment:
instead of returning a string here, return a tuple like so:

https://github.com/status-im/nim-beacon-chain/blob/e1fb18d7dbb5c40e5520f06a71196b31827487ac/beacon_chain/spec/datatypes.nim#L615